cfme/tests/configure/test_visual_cloud.py

```diff
@@ -38,53 +38,49 @@
 }
 
 
-@pytest.fixture(scope="module")
-def visual(appliance):
-    return Visual(appliance=appliance)
-
-
-@pytest.yield_fixture(scope="module")
-def set_grid(visual):
-    gridlimit = visual.grid_view_limit
-    visual.grid_view_limit = 5
+@pytest.yield_fixture(scope='module')
+def set_grid(appliance):
+    gridlimit = appliance.user.my_settings.visual.grid_view_limit
+    appliance.user.my_settings.visual.grid_view_limit = 5
     yield
-    visual.grid_view_limit = gridlimit
+    appliance.user.my_settings.visual.grid_view_limit = gridlimit
 
 
-@pytest.yield_fixture(scope="module")
-def set_tile(visual):
-    tilelimit = visual.tile_view_limit
-    visual.tile_view_limit = 5
+@pytest.yield_fixture(scope='module')
+def set_tile(appliance):
+    tilelimit = appliance.user.my_settings.visual.tile_view_limit
+    appliance.user.my_settings.visual.tile_view_limit = 5
     yield
-    visual.tile_view_limit = tilelimit
+    appliance.user.my_settings.visual.tile_view_limit = tilelimit
 
 
-@pytest.yield_fixture(scope="module")
-def set_list(visual):
-    listlimit = visual.list_view_limit
-    visual.list_view_limit = 5
+@pytest.yield_fixture(scope='module')
+def set_list(appliance):
+    listlimit = appliance.user.my_settings.visual.list_view_limit
+    appliance.user.my_settings.visual.list_view_limit = 5
     yield
-    visual.list_view_limit = listlimit
+    appliance.user.my_settings.visual.list_view_limit = listlimit
 
 
-def set_default_page(visual):
-    visual.login_page = "Cloud Intel / Dashboard"
+def set_default_page():
+    appliance = current_appliance()
```

Comment:
**Required**: Use appliance here, can be passed as parameter, or function be converted to fixture